com.ibm.wala.util/src/com/ibm/wala/util/intset/MultiModalIntVector.java

```diff
@@ -81,8 +81,8 @@ float getGrowthFactor() {
    */
   float getGrowthFactor(int size) {
     if (CURRENT_GROWTH_RATE >= MINIMUM_GROWTH_FACTOR) {
```

Comment:
Since we're editing this code, can we rename `CURRENT_GROWTH_RATE` to `currentGrowthRate`, to make it clearer it's an instance variable?

Reply:
Renamed.